tracks/scala/leap/mentoring.md

```diff
@@ -0,0 +1,25 @@
+### Reasonable solutions
+
+```scala
```

Comment:
Just a suggestion. In the C# track, I use a sub-header to describe the various reasonable solutions if there is more than one (what makes them unique). For an example, see [the notes](https://github.com/exercism/website-copy/blob/master/tracks/csharp/exercises/bob/mentoring.md). You could consider adding those, but feel free to ignore.

Reply:
Thanks!